packages/fs-detectors/src/monorepos/get-monorepo-default-settings.ts

```diff
@@ -21,12 +21,19 @@ export class MissingBuildTarget extends Error {
   }
 }
 
+type MonorepoDefaultSettings = {
+  buildCommand: string | null;
+  installCommand: string | null;
```

Comment:
Should we also allow `undefined` just in case?

Reply:
Yeah I am good with having that for backup 👍